openff/qcsubmit/results/caching.py

```diff
@@ -102,11 +102,11 @@ def _cached_client_query(
         if (client_address, query_id) in query_cache
     ]
 
-    client = _cached_fractal_client(client_address)
+    client = cached_fractal_client(client_address)
 
     logger.debug(f"starting {query_name} to {client_address}")
 
-    batch_query_ids = _batched_indices(missing_query_ids, client.query_limit)
+    batch_query_ids = batched_indices(missing_query_ids, client.query_limit)
```

Comment:
Just wondering what the names have changed?

Reply:
I'm starting to access these a bit outside of the `caching` module so I thought it was probably ok to just make them public members. I'd be happy to make them private though if that's your preference?